<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FORM_ControlPort4</class>
 <widget class="QWidget" name="FORM_ControlPort4">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">/*background-image: url(:/pic_de260/bg_01.png);*/
QLabel{
color: rgb(15, 175, 252);
font: 17pt &quot;WenQuanYi Zen Hei&quot;;
}</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>930</x>
     <y>50</y>
     <width>371</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>TakaoPGothic</family>
     <pointsize>30</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(252, 233, 79);
font: 30pt &quot;TakaoPGothic&quot;;</string>
   </property>
   <property name="text">
    <string>控制器I/O</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="portinfo">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>130</y>
     <width>371</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>WenQuanYi Zen Hei</family>
     <pointsize>20</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font: 20pt &quot;WenQuanYi Zen Hei&quot;;</string>
   </property>
   <property name="text">
    <string>液压控制器DI端口查询</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="pageinfo">
   <property name="geometry">
    <rect>
     <x>1420</x>
     <y>130</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(238, 238, 236);
font: 20pt &quot;WenQuanYi Zen Hei&quot;;</string>
   </property>
   <property name="text">
    <string>第4/6页</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>1290</x>
     <y>50</y>
     <width>81</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>WenQuanYi Zen Hei</family>
     <pointsize>17</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="resources.qrc">:/ico_260/915.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="text8">
   <property name="geometry">
    <rect>
     <x>816</x>
     <y>452</y>
     <width>345</width>
     <height>35</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>345</width>
     <height>30</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>WenQuanYi Zen Hei</family>
     <pointsize>17</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;DI14-高压过滤器堵塞报警(前)&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="light_2">
   <property name="geometry">
    <rect>
     <x>1167</x>
     <y>247</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>30</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="resources.qrc">:/pic_de260/ico_25.png</pixmap>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="text4">
   <property name="geometry">
    <rect>
     <x>816</x>
     <y>288</y>
     <width>345</width>
     <height>35</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>345</width>
     <height>30</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>WenQuanYi Zen Hei</family>
     <pointsize>17</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>DI4-主驾安全带检测</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="light_4">
   <property name="geometry">
    <rect>
     <x>1167</x>
     <y>288</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>30</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="resources.qrc">:/pic_de260/ico_25.png</pixmap>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="text6">
   <property name="geometry">
    <rect>
     <x>816</x>
     <y>329</y>
     <width>345</width>
     <height>35</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>345</width>
     <height>30</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>WenQuanYi Zen Hei</family>
     <pointsize>17</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>DI9-驾驶室门检测</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="light_6">
   <property name="geometry">
    <rect>
     <x>1167</x>
     <y>329</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>30</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="resources.qrc">:/pic_de260/ico_25.png</pixmap>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="text7">
   <property name="geometry">
    <rect>
     <x>816</x>
     <y>370</y>
     <width>345</width>
     <height>35</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>345</width>
     <height>30</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>WenQuanYi Zen Hei</family>
     <pointsize>17</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;DI12-回油过滤器堵塞报警(前)&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="light_7">
   <property name="geometry">
    <rect>
     <x>1167</x>
     <y>370</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>30</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="resources.qrc">:/pic_de260/ico_25.png</pixmap>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="text9">
   <property name="geometry">
    <rect>
     <x>816</x>
     <y>411</y>
     <width>345</width>
     <height>35</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>345</width>
     <height>30</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>WenQuanYi Zen Hei</family>
     <pointsize>17</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;DI13-回油过滤器堵塞报警(后)&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="light_8">
   <property name="geometry">
    <rect>
     <x>1167</x>
     <y>411</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>30</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="resources.qrc">:/pic_de260/ico_25.png</pixmap>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="light_9">
   <property name="geometry">
    <rect>
     <x>1167</x>
     <y>452</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>30</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="resources.qrc">:/pic_de260/ico_25.png</pixmap>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="text1">
   <property name="geometry">
    <rect>
     <x>816</x>
     <y>206</y>
     <width>345</width>
     <height>35</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>345</width>
     <height>30</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>WenQuanYi Zen Hei</family>
     <pointsize>17</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>DI1-接近开关（前）</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="light_1">
   <property name="geometry">
    <rect>
     <x>1167</x>
     <y>206</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>30</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="resources.qrc">:/pic_de260/ico_25.png</pixmap>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="text2">
   <property name="geometry">
    <rect>
     <x>816</x>
     <y>247</y>
     <width>345</width>
     <height>35</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>345</width>
     <height>30</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>WenQuanYi Zen Hei</family>
     <pointsize>17</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>DI2-接近开关（后）</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="text7_2">
   <property name="geometry">
    <rect>
     <x>1226</x>
     <y>370</y>
     <width>390</width>
     <height>35</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>390</width>
     <height>30</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>WenQuanYi Zen Hei</family>
     <pointsize>17</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;DI22-制动蓄能力预冲压力低报警(右)&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="light_17">
   <property name="geometry">
    <rect>
     <x>1622</x>
     <y>370</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>30</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="resources.qrc">:/pic_de260/ico_25.png</pixmap>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="light_14">
   <property name="geometry">
    <rect>
     <x>1167</x>
     <y>534</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>30</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="resources.qrc">:/pic_de260/ico_25.png</pixmap>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="text5_2">
   <property name="geometry">
    <rect>
     <x>1226</x>
     <y>288</y>
     <width>390</width>
     <height>35</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>390</width>
     <height>30</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>WenQuanYi Zen Hei</family>
     <pointsize>17</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>DI20-制动蓄能器预冲压力低报警(左)</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="light_15">
   <property name="geometry">
    <rect>
     <x>1622</x>
     <y>288</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>30</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="resources.qrc">:/pic_de260/ico_25.png</pixmap>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="text6_2">
   <property name="geometry">
    <rect>
     <x>1226</x>
     <y>329</y>
     <width>390</width>
     <height>35</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>390</width>
     <height>30</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>WenQuanYi Zen Hei</family>
     <pointsize>17</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>DI21-制动蓄能器预冲压力低报警(中)</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="light_16">
   <property name="geometry">
    <rect>
     <x>1622</x>
     <y>329</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>30</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="resources.qrc">:/pic_de260/ico_25.png</pixmap>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="text3_2">
   <property name="geometry">
    <rect>
     <x>816</x>
     <y>493</y>
     <width>345</width>
     <height>35</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>345</width>
     <height>30</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>WenQuanYi Zen Hei</family>
     <pointsize>17</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>DI15-高压过滤器堵塞报警(中)</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="light_13">
   <property name="geometry">
    <rect>
     <x>1167</x>
     <y>493</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>30</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="resources.qrc">:/pic_de260/ico_25.png</pixmap>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="text4_2">
   <property name="geometry">
    <rect>
     <x>816</x>
     <y>534</y>
     <width>345</width>
     <height>35</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>345</width>
     <height>30</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>WenQuanYi Zen Hei</family>
     <pointsize>17</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>DI16-高压过滤器堵塞报警(后)</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="light_12">
   <property name="geometry">
    <rect>
     <x>1622</x>
     <y>247</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>30</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="resources.qrc">:/pic_de260/ico_25.png</pixmap>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="text1_2">
   <property name="geometry">
    <rect>
     <x>1226</x>
     <y>206</y>
     <width>390</width>
     <height>35</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>390</width>
     <height>30</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>WenQuanYi Zen Hei</family>
     <pointsize>17</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>DI17-转向蓄能器预冲压力低报警1</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="light_11">
   <property name="geometry">
    <rect>
     <x>1622</x>
     <y>206</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>30</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="resources.qrc">:/pic_de260/ico_25.png</pixmap>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="text2_2">
   <property name="geometry">
    <rect>
     <x>1226</x>
     <y>247</y>
     <width>390</width>
     <height>35</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>390</width>
     <height>30</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>WenQuanYi Zen Hei</family>
     <pointsize>17</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>DI18-转向蓄能器预冲压力低报警2</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
